src/ResourceManager/Storage/Commands.Management.Storage/StorageAccount/StorageManagementClient.cs

```diff
@@ -17,7 +17,7 @@
 using Microsoft.Azure.Management.Storage;
 using System;
 
-namespace Microsoft.Azure.Commands.Compute
+namespace Microsoft.Azure.Commands.Management.Storage
```

Comment:
You mean other options for namespace? We decided to use this namespace after discussed with Mark and according to his suggestion..